cumulusci/tasks/github/tests/test_release.py

```diff
@@ -83,6 +83,7 @@ def test_run_task(self):
                         "version": "1.0",
                         "version_id": "04t000000000000",
                         "dependencies": [{"namespace": "foo", "version": "1.0"}],
+                        "package_type": "1GP",
```

Comment:
We're getting free code coverage because of how the logic is structured; could we add an assert somewhere to validate the generated tag message?

Reply:
good call. I've gone ahead and added an assert on the request body to ensure the expected package type is set.